wordpress-mobile/gutenberg-mobile
# Fix/colors and richtextinputevent

Fixes 2 issues introduced recently:

1. `Error: colors.scss could not be resolved`
2. The renamed UnstableRichTextInputEvent (on the web) that was still RichTextInputEvent on mobile

To test:

1. Run the demo app. Is should open without any Metro errors
2. Tap in a rich-text block. It should just work as normal
